benchmarks/README.rst

```diff
@@ -36,6 +36,14 @@ defined in ``benchmarks/``. (Note: this could take a while. Each
 benchmark is run multiple times to measure the distribution in
 execution times.)
 
+For **testing** benchmarks locally, it may be better to run these without
+replications::
+
+    cd benchmarks/
+    asv run -n -e --python=same -q -b $REGEXP
+
+Where the regular expression used to match benchmarks is stored in ``$REGEXP``.
```

Comment:
```suggestion
    cd benchmarks/
    export REGEXP="bench.*Ufunc"
    asv run --dry-run --show-stderr --python=same --quick -b $REGEXP

Where the regular expression used to match benchmarks is stored in ``$REGEXP``, and `--quick` is used to avoid repititions.
```